api/app/Console/Commands/SendNotificationsPoolPublished.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Enums\NotificationFamily;
+use App\Enums\PublishingGroup;
+use App\Models\Pool;
+use App\Models\User;
+use App\Notifications\NewJobPosted;
+use Illuminate\Console\Command;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Log;
+use Throwable;
+
+class SendNotificationsPoolPublished extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'send-notifications:pool-published';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send notifications to users about a new pool being published.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        // workaround until we get better logging in prod #11289
+        $onDemandLog = Log::build([
+            'driver' => 'single',
+            'path' => App::isProduction() // workaround for storage_path misconfigured in prod #11471
+                ? '/tmp/api/storage/logs/jobs.log'
+                : storage_path('logs/jobs.log'),
+        ]);
+
+        $this->info('SendNotificationsPoolPublished running at '.Carbon::now()->toDateTimeString().'.');
+
+        $successCount = 0;
+        $failureCount = 0;
+
+        $endOfSpan = Carbon::now();
+        $startOfSpan = Carbon::now()->subHours(24); // assuming the last reporting job ran about 24 hours ago
+        $this->info("Finding pools published between $startOfSpan and $endOfSpan.");
+
+        $poolsPublishedRecently = Pool::query()
```

Comment:
one more question .. this pr checks the pools published within 24 hours but doesn't check for open pools. 
In the use case , they have published something accidentally and they closed it immediately, we would still send notification for that closed pool.

I'm wondering should we add that check for only active pools?

Reply:
Good catch.  If I had remembered to update the tests first, this would have been caught.  :facepalm: